src/components/projects/CategorisedKeywordsInput.tsx

```diff
@@ -316,7 +316,7 @@ export const CategorisedKeywordsInput: React.FC<{
       <OptionWrapper
         focused={keywordToEdit ? data._id === keywordToEdit._id : false}
         ref={innerRef}
-        {...innerProps}
+        {...(innerProps as any)}
```

Comment:
any chance we can do a more specific type here?

Reply:
updated the ref to be null to ignore the ref of innerProps so with that no need for the `any`. 

and I used it as there is a type mismatch with the ref of styled-components dependency and the one we get from `react-select`. in the new version of [react-select](https://github.com/JedWatson/react-select/blob/master/docs/styled-components.tsx) they remove that dependency and use inline CSS to style custom component!